overlord/ifacestate/ifacemgr.go

```diff
@@ -96,9 +105,33 @@ func Manager(s *state.State, hookManager *hookstate.HookManager, runner *state.T
 
 // Ensure implements StateManager.Ensure.
 func (m *InterfaceManager) Ensure() error {
+	if m.udevMon != nil || m.udevMonitorDisabled {
+		return nil
+	}
+
+	// retry udev monitor initialization every 5 minutes
+	now := time.Now()
+	if now.After(m.udevInitTime) {
+		err := m.initUdevMonitor()
+		if err != nil {
+			m.udevInitTime = now.Add(udevInitRetryTimeout)
```

Comment:
This should probably be named udevRetryTimeout or similar. InitTime makes it sound like it was the time it got initialized, but it's the other end of it.